### ISSUE TITLE
Preservation Storage Location to LTO

### ISSUE THREAD

**Issue by Muraszko** [WGBH/hydradam#6]
Currently, the options for copying and storing the source files ingested into HydraDAM are locally (the server in which HydraDAM is installed on) and via HSM (that is tricking HydraDAM into thinking it's locally connected).

For our purposes, we need to ingest from a source hard drive and copy those files onto the locally connected LTO 6 drive, all while retaining the folder structure of the source drive.

![screen shot 2014-07-16 at 9 33 52 am](https://cloud.githubusercontent.com/assets/6650285/3599362/db671e78-0ced-11e4-81c1-19f6b4fba1d7.jpg)

DONE WHEN:
- User ingests file(s) using HydraDAM and copies of those files are sent to a locally connected LTO 6 deck.

- HydraDAM records show on which LTO tape the files have been copied to.

**Comment by afred** [comment]
Closing due to change in workflow. We decided for now, that copying drives to LTO should happen external to HydraDAM, and that ingesting of digital files should happen either from mounted harddrive, mounted LTO tape, or from the local filesystem.

This highlights the need to be able to associate multiple locations of a digital file after it has been initially ingested (using `pbcoreInsantiation` metadata perhaps?). But that will probably be comprised of multiple tickets, later down the road.

@Muraszko ^^ does all that sound about right?
